src/Containers.jsx

```diff
@@ -79,14 +187,13 @@ class Containers extends React.Component {
 
         let startStopActions = [];
         if (isRunning)
-            startStopActions.push({ label: _("Stop"), onActivate: () => this.stopContainer(container)});
+            startStopActions.push({ label: _("Stop"), onActivate: () => this.stopContainer(container, undefined)});
         else
             startStopActions.push({ label: _("Start"), onActivate: () => this.startContainer(container)});
 
         startStopActions.push({
             label: _("Restart"),
-            // onActivate: this.restartContainer,
-            onActivate: this.restartContainer,
+            onActivate: this.restartContainer(container, undefined),
```

Comment:
`restartContainer()` does not take a second argument. Also, you don't actually need to pass trailing `undefined` arguments, they will be undefined by default.